src/service/resources-impl.js

```diff
@@ -387,15 +388,40 @@ export class Resources {
     this.resources_.push(resource);
 
     if (this.isRuntimeOn_) {
-      // Try to immediately build element, it may already be ready.
-      resource.build(this.forceBuild_);
-      this.schedulePass();
+      if (this.documentReady_) {
```

Comment:
This seems like a good idea, the only thing is we would't be calling `buildReadyResources_` until a non-parsed element appear or documentReady kicks off, I think this might delay some of the elements that were already added to `pendingBuildResources`. No?